_layouts/blog_by_tag.html

```diff
@@ -9,21 +9,55 @@
+            <div class="col {% if filename == 'urgent' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/urgent.html">{% t blog.urgent %}</a></div>
+            <div class="col {% if filename == 'releases' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/releases.html">{% t blog.releases %}</a></div>
+            <div class="col {% if filename == 'community' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/community.html">{% t blog.community %}</a></div>
+            <div class="col {% if filename == 'dev diaries' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/dev%20diaries.html">{% t blog.meetinglogs %}</a></div>
+        </div>
+    </div>
+  <!-- End category selector: desktop -->
+  <!-- Category selector: mobile -->
+  <div class="container full">
+      <div class="info-block row center-xs" id="pick-platform">
+          <div class="mob dropdowndrop">
+              <input id="filter" type="checkbox" name="category-filter"/>
+              <label for="filter">{% t blog.filter %}</label>
+              <ul id="menu">
+                  <li><a href="{{ site.baseurl_root }}/blog">{% t blog.allposts %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/urgent.html">{% t blog.urgent %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/releases.html">{% t blog.releases %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/community.html">{% t blog.community %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/dev%20diaries.html">{% t blog.meetinglogs %}</a></li>
+              </ul>
+          </div>
+      </div>
+  </div>
+  <!-- End category selector: mobile -->
+</div>
+
 <div class="site-wrap">
     <section class="container">
             <div class="row">
                 <!-- left two-thirds block-->
                 <div class="left two-thirds col-lg-8 col-md-8 col-sm-8 col-xs-12">
-                    <div class="info-block">
+                  <div class="info-block">
                         <div class="row center-xs">
-                        <div class="page-title">
-		                  <h2 class="inline"><span class="kicks">{{ tag.name }}</span></h2>
-		                </div>
-                    </div>
-                    
+                          <div class="page-title">
+                          {% if tag.name != "Dev Diaries" %}
+                            <h2 class="inline"><span class="kicks">{{ tag.name }}</span></h2>
+                          {% else %}
+                            <h2 class="inline"><span class="kicks">{% t blog.meetinglogs %}</span></h2>
+                          {% endif %}
+		                      </div>
+                        </div>
                        {% if site.tags[tag.slug] %}
-				{% for post in site.tags[tag.slug] %}
-                  <div class="post-tag">
+			                 	 {% for post in site.tags[tag.slug] %}
```

Comment:
Here too, beneath and above are all spaces so this should be all spaces too.

Reply:
Should be fixed now. Please let me know if anything else looks off.